src/EditorFeatures/CSharpTest/AddParameterCheck/AddParameterCheckTests.cs

```diff
@@ -0,0 +1,579 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
I recommend looking at this file to get a sense of all the cases this feature handles.